mod/ckeditor/elgg-plugin.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+	'home_language_keys' => [
+		'ckeditor:html',
+		'ckeditor:visual',
+	],
+];
```

Comment:
General container for plugin declarative configuration. Or we can do individual files for each setting we come up with...